charts/jenkins/README.md

```diff
@@ -415,6 +416,35 @@ $ helm install my-release -f values.yaml stable/jenkins
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+#### Defining Caches for your agent pods
+
+Its possible to define a set of cache volumes that can be mounted by agent pods for storing libraries and other artifacts for build. The volume defined, under `persistence`, will be created as a new `PVC` by the chart. The volume has to be mounted under the `volumes` section. Along with the cache, a `CronJob` can be defined under the `clear` section to perform cache activies, including clearing old libraries, on a periodic basis. The `CronJob` is required both to prevent unused libraries from accumulating over time and to avoid security issues.
+
+> **Tip**: Ensure a PVC and CronJob with a given `componentName` is defined only once
```

Comment:
Do we have a chance to check this? If so then we could fail template rendering with an error message.

Reply:
I have added the `idleminutes` note, however, the cache is `Read Write Many` so is mounted on multiple agents as required. It makes sense to include the note.

I have modified the `cache-cronjob` file to fail with an error message, however, the error message coming from helm is probably clearer. Can modify `cache-pvc` too if this looks OK, else we can go with the default error thrown.